src/ADAL.PCL/BrokerEvent.cs

```diff
@@ -38,6 +38,8 @@ internal class BrokerEvent : DefaultEvent
         internal BrokerEvent(string eventName) : base(eventName)
         {
             //Fill in the default parameters
+            BrokerApp = "AuthenticatorApp";
+            BrokerVersion = "2";
```

Comment:
This is the maximum version that ADAL talks to and is hard coded as "2".Hence I used the same as default value to provide the best estimate.